src/api/python/cvc5.pxi

```diff
@@ -1010,11 +1125,13 @@ cdef class Solver:
     @expand_list_arg(num_req_args=0)
     def mkOp(self, k, *args):
         """
-        Supports the following uses:
+            Create operator.
 
-        - ``Op mkOp(Kind kind)``
-        - ``Op mkOp(Kind kind, const string& arg)``
-        - ``Op mkOp(Kind kind, uint32_t arg0, ...)``
+            Supports the following arguments:
```

Comment:
Here, I think the `expand_list_arg` is not properly used: the argument should be 1, not 0. I suspect calling `mkOp(kind, [1,2,3,4])` will throw an exception while `mkOp([kind, 1,2,3,4])` will work fine.